sph/include/sph/positions.hpp

```diff
@@ -38,6 +38,41 @@
 
 namespace sph
 {
+template<class T>
+T fbcCheck(T x, T y, T z, T vx, T vy, T vz, T h, const cstone::Box<T> box)
```

Comment:
take box by const-reference
doc-string missing

Reply:
This function should just return a bool whether the particle is inside the fixed zone.
If that's the case, the rest of the integrator is simply skipped for this particle (the `continue` keyword)